version.go

```diff
@@ -263,3 +330,14 @@ func comparePrePart(s, o string) int {
 	}
 	return -1
 }
+
+func areEq(v1, v2 *Version) bool {
```

Comment:
Note, This should be moved to the range.go file since that's where it's used. This can always happen after this pull request is merged but I wanted to document it.